test/tests/selftest/pbs_json_test_report.py

```diff
@@ -118,6 +127,10 @@ def test_json_fields(self):
             for t in jdata['testsuites'][s]:
                 for u in jdata['testsuites'][s]['testcases']:
                     for r in verify_data['test_cases_info']:
+                        if r == 'requirements':
```

Comment:
This loop previously checked only the keys of test case information. I suggest to move this field value check to 'for' loop in line 142 "for k, v in field_values.items():"; since that is the place where the test checks for other field values - machine_info and test result status.